packages/app-project/next.config.js

```diff
@@ -11,7 +11,7 @@ const talkHosts = require('./config/talkHosts')
 const assetPrefixes = {
   development: '/projects',
   branch: 'https://fe-project-branch.preview.zooniverse.org/projects',
-  staging: 'https://fe-project.preview.zooniverse.org/projects',
+  staging: 'https://frontend.preview.zooniverse.org/projects',
```

Comment:
You can actually delete this setting, since this duplicates the URL of the NextJS app. I don’t know why I’ve never noticed that before. You only need to set `assetPrefix` when your app and its assets are hosted on different domains.

> Note: Next.js 9.5+ added support for a customizable [Base Path](https://nextjs.org/docs/api-reference/next.config.js/basepath), which is better suited for hosting your application on a sub-path like /docs. We do not suggest you use a custom Asset Prefix for this use case.

https://nextjs.org/docs/api-reference/next.config.js/cdn-support-with-asset-prefix

Reply:
```suggestion
```

Reply:
#4240 removes `assetPrefix` from both apps where it's been used as a replacement for `basePath`.

Reply:
Ah interesting! I think I understand. So once the presumed CDN issue is revolved / we want production site traffic to go through the CDN again, we can also remove  `production : 'https://fe-project.zooniverse.org/projects' from `assetPrefixes`.